galaxychop/core.py

```diff
@@ -849,4 +846,91 @@ def paramcirc(self):
         eps_[neg_star[fin_star[mask]]] = eps[mask]
         eps_r_[neg_star[fin_star[mask]]] = eps_r[mask]
 
-        return E_star_, eps_, eps_r_
+        return (E_star_, eps_, eps_r_)
+
+    def values(self, star=True):
+        """
+        2D and 1D inputs converter.
+
+        Builds two arrays: one 2D listing all the parameters of each
+        particle and other 1D showing whether the particle is a star,
+        gas or dark matter one.
+
+        Parameters
+        ----------
+        star : `bool`. Default=False
+            Indicates if the stars particles are going to be use to
+            build the array
+        dm : `bool`. Default=False
+            Indicates if the dark matter particles are going to be use to
+            build the array
+        gas : `bool`. Default=False
+            Indicates if the gas particles are going to be use to
+            build the array
+
+
+        Returns
+        -------
+        X : `np.ndarray(n,7)`
+            2D array where each file it is a diferent particle and
+            each column is a parameter of the particles (E_star, eps, eps_r)
+        y : `np.ndarray(n)`
+            1D array where is identified the nature of each particle
+            0=star, 1=gas and 2=dark matter
+        """
```

Comment:
If you look at what this method does, it does not correspond to the documentation provided.